dao/src/main/java/org/thingsboard/server/dao/resource/TbResourceInfoDao.java

```diff
@@ -24,8 +25,8 @@
 
 public interface TbResourceInfoDao extends Dao<TbResourceInfo> {
 
-    PageData<TbResourceInfo> findAllTenantResourcesByTenantId(UUID tenantId, PageLink pageLink);
+    PageData<TbResourceInfo> findAllTenantResourcesByTenantId(TbResourceInfoFilter tbResourceInfoFilter, PageLink pageLink);
```

Comment:
it is better to use simple variable name that is more readable (just ```filter``` instead of ```tbResourceInfoFilter```)